prime-router/src/main/kotlin/tokens/AuthenticatedClaims.kt

```diff
@@ -20,7 +20,7 @@ class AuthenticatedClaims {
     /** Name of user as extracted from the subject claim.  Usually an email address */
     val userName: String
     /** Does this user have the prime administrator claim? */
-    val isPrimeAdmin: Boolean
+    var isPrimeAdmin: Boolean
```

Comment:
I have a TODO item for this in a unit test I wrote below, and intend to revert this before merging in. See below.

Reply:
fixed it